apps/website/app/(docs)/docs/obsidian/pages/canvas.md

```diff
@@ -0,0 +1,125 @@
+---
+title: "Canvas"
+date: "2025-01-01"
+author: ""
+published: true
+---
+The Canvas feature in Discourse Graph provides an interactive visual workspace for creating and connecting Discourse Nodes and Relations. Built on top of tldraw, it allows you to visually map out discourse structures with drag-and-drop functionality, adding other visual elements like texts, scribbles, embeddings, and images.
+
+## Creating a New Canvas
+
+### Using the Command Palette
+
+1. Open the Command Palette (`Cmd/Ctrl + P`)
+2. Search for "Create new Discourse Graph canvas"
+3. Press Enter to create a new canvas
+
+![](https://firebasestorage.googleapis.com/v0/b/firescript-577a2.appspot.com/o/imgs%2Fapp%2Fdiscourse-graphs%2F_uF6WGGgga.png?alt=media&token=4fd61b3c-3a50-48bb-b034-b5e0a8156c8f)
+The canvas will be created in your configured canvas folder (default: `Discourse Canvas/`) with a timestamp-based filename like `Canvas-2024-01-15-1430.md`.
+
+## Opening and Viewing Canvas
+
+### Opening a Canvas
+1. Open the file of the canvas
+3. In case the file is still in markdown format, you can open the canvas mode by either
+
+Using the command pallete
+
+![](https://firebasestorage.googleapis.com/v0/b/firescript-577a2.appspot.com/o/imgs%2Fapp%2Fdiscourse-graphs%2FxhxReS4Ylp.png?alt=media&token=f04ec9da-5282-45fa-b989-60f08ea0b8d6)
+Or click on the Canvas icon at the top right corner
+![](https://firebasestorage.googleapis.com/v0/b/firescript-577a2.appspot.com/o/imgs%2Fapp%2Fdiscourse-graphs%2FDziMHaUNq9.png?alt=media&token=85e2ed2d-01b0-49ac-ae05-a34e20218a5a)
+
+
+***Warning*: DO NOT MODIFY MARKDOWN CONTENT OF FILE**
+
+## Adding Nodes to Canvas
+
+### Creating New Discourse Nodes
+
+1. **Using the Discourse Node Tool**:
```

Comment:
nit: only use numbers when steps must be followed in sequence or when order conveys meaning